src/main/kotlin/racing/domain/car/Car.kt

```diff
@@ -0,0 +1,29 @@
+package racing.domain.car
+
+import racing.domain.utils.RandomNumberGenerator
+
+data class Car(
+    private val _carName: String,
+    private var _moveCount: Int = 0,
+) {
+
+    val carName: String
+        get() { return _carName }
+
+    val moveCount: Int
+        get() { return _moveCount }
```

Comment:
```suggestion
    val carName: String
        get() = _carName

    val moveCount: Int
        get() = _moveCount
```
return문 없이도 다음과 같이 작성할 수 있습니다!

Reply:
코틀린스러운 코드로 변경했습니다-!